mkt/__init__.py

```diff
@@ -1,18 +1,18 @@
 import threading
 
 from mkt.constants import (categories, comm, platforms, iarc_mappings,
-                           ratingsbodies)
-from mkt.constants.applications import *
-from mkt.constants.base import *
-from mkt.constants.payments import *
-from mkt.constants.platforms import *
-from mkt.constants.search import *
-from mkt.constants.submit import *
+                           ratingsbodies)  # flake8: noqa
+from mkt.constants.applications import *  # flake8: noqa
+from mkt.constants.base import *  # flake8: noqa
+from mkt.constants.payments import *  # flake8: noqa
+from mkt.constants.platforms import *  # flake8: noqa
+from mkt.constants.search import *  # flake8: noqa
+from mkt.constants.submit import *  # flake8: noqa
```

Comment:
Yeah, it would be nice if there's a way to allow `import *` in an `__init__.py` file.